eng/Signing.props

```diff
@@ -5,7 +5,6 @@
   <ItemGroup>
     <!-- Reset Arcade's defaults. -->
     <ItemsToSign Remove="@(ItemsToSign)" />
-    <StrongNameSignInfo Remove="@(StrongNameSignInfo)" />
```

Comment:
Think we did this to remove all of Arcade's defaults because they conflicted w/ what we wanted to sign. But, I'm fine leaving the defaults around if they don't cause problems (anymore).

Reply:
This is basically what we were discussing last Friday. The defaults in arcade what we want in post-build and in-build.